grl/mdp.py

```diff
@@ -278,7 +287,8 @@ def observation_space(self):
     def is_abstract_policy(self, pi):
         agg_states = (self.phi.sum(axis=0) > 1)
         for idx, is_agg in enumerate(agg_states):
-            agg_cluster = (one_hot(idx, self.n_obs) @ self.phi.transpose()).astype(bool)
+            agg_cluster = (
+                one_hot(idx, self.observation_space.n) @ self.phi.transpose()).astype(bool)
```

Comment:
What's this `agg_cluster` thing? I'm assuming agg means aggregated, but that's where I'm stuck.

Reply:
So the git blame says I wrote this line, but I don't think I was the one to write this?

Reply:
might just be the `self.observation_space.n` change - did you do a global refactor? might have put your name on a bunch on lines that aren't yours